hazelcast-tpc-engine/src/main/java/com/hazelcast/internal/tpc/util/ExceptionUtil.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2008-2022, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.internal.tpc.util;
+
+public final class ExceptionUtil {
+
+    private ExceptionUtil() {
+    }
+
+    /**
+     * Ignores the given exception.
+     *
+     * @param t the exception to ignore
+     */
+    public static void ignore(Throwable t) {
```

Comment:
Quite risky ;)

If you want to ignore some exception that is not exceptional (thank you Oracle for tons of exception-driven flow), then you can name the variable `ignored` and IntelliJ won't report problem with it

Reply:
In the past checkstyle complained about doing this.

Reply:
Ah ok, it's probably disabled for test classes.

I see in code we already have `EmptyStatement.ignore`